src/Stratis.Bitcoin.Features.Api/ApiSettings.cs

```diff
@@ -52,13 +52,6 @@ public class ApiSettings
         /// <summary>Use HTTPS or not.</summary>
         public bool UseHttps { get; set; }
     
-        /// <summary>
```

Comment:
I would have been happier with 
```
public ApiSettings(Network network) : this(NodeSettings.Default(network))	
{
}
```
but this is fine, we don't use these overload apart from in the tests, and having a more explicit call is not detrimental in any way.